test/functional/options/tabline_spec.lua

```diff
@@ -0,0 +1,40 @@
+local helpers = require('test.functional.helpers')(after_each)
```

Comment:
move this to `test/functional/ui/tabline_spec.lua`, it has basically nothing to do with options

Reply:
Ok I thought that file was for external UI tests xD